src/components/MultiGestureCanvas/index.tsx

```diff
@@ -235,16 +236,12 @@ function MultiGestureCanvas({
         };
     });
 
+    const containerStyles = useMemo(() => [styles.flex1, StyleUtils.getMultiGestureCanvasContainerStyle(canvasSize.width)], [StyleUtils, canvasSize.width, styles.flex1]);
```

Comment:
This is not correct. The reference of `styles` will always change when the theme changes. Technically, the particular `styles.flex1` style will not change, but i wouldn't remove it from the memo deps anyway. It will only trigger an execution if `styles` changes anyway, so there's no benefit in removing it from the list...